test/test_kND.py

```diff
@@ -0,0 +1,48 @@
+from unittest import TestCase
+from pandas import DataFrame
+from pandas import Series
+from numpy import array
+
+from dna.kND import KNearestDatasets
+
+
+class KNearestDatasetsTestCase(TestCase):
+
+    def test_knn_regression(self):
+        d1 = 'dataset_1'
+        mfs1 = [1.0, 1.0, 1.0]
+        d2 = 'dataset_2'
+        mfs2 = [2.0, 2.0, 2.0]
+        p1 = 'pipeline_1'
+        p2 = 'pipeline_2'
+        p3 = 'pipeline_3'
+
+        d1p1 = 0.9
+        d1p2 = 0.8
+        d1p3 = 0.7
+        d2p1 = 0.7
+        d2p2 = 0.6
+        d2p3 = 0.5
```

Comment:
Could you use a list here?

Reply:
I can't because I need them to be separated whn I add them to the dictionaries on line 8. Those are the dictionaries that I make the data frames from.